src/templates/next/components/internal/Footer/Footer.tsx

```diff
@@ -46,15 +46,15 @@ const FooterItem = ({
         href={url}
         target="_blank"
         rel="noopener noreferrer nofollow"
-        className="flex gap-1 items-center w-fit hover:underline hover:underline-offset-2"
+        className="flex gap-1 items-center w-fit hover:underline hover:underline-offset-2 line-clamp-1"
       >
         {title}
-        <BiLinkExternal className="w-[0.875rem] lg:w-[1.5rem] h-auto flex-shrink-0" />
+        <BiLinkExternal className="w-[0.875rem] lg:w-[1rem] h-auto flex-shrink-0" />
```

Comment:
nit: can use `lg:w-4` instead!

Reply:
fixed in [19ad69d](https://github.com/isomerpages/isomer-components/pull/100/commits/19ad69d3762c74d88026a9132d6381c0f9ef2d48)!